src/System.Runtime.Serialization.Xml/tests/SerializationTestTypes/DCRTypeLibrary.cs

```diff
@@ -167,4 +169,29 @@ public virtual object[] DataContractResolverTypes
             set { dataContractResolverTypes = value; }
         }
     }
+
+    [DataContract]
+    public class DefaultCollections
+    {
+        [DataMember]
+        ArrayList arrayList = new ArrayList() { new Person() };
```

Comment:
Use `private` explicitly for private members. The name of private field starts with "_".